lib/feed.xml

```diff
@@ -33,7 +33,9 @@
       <updated>{{ post.date | date_to_xmlschema }}</updated>
 
       <id>{{ post.id | prepend: url_base | xml_escape }}</id>
-      <content type="html">{{ post.content | markdownify | xml_escape }}</content>
+      <content type="html" xml:base="{{ url_base | xml_escape }}">
```

Comment:
I think it should be `post.url` rather than just `url_base`, since that is what links would be relative to.